From be925e84edcecd879e953bdb68c10f98825dba53 Mon Sep 17 00:00:00 2001
From: Shwetha K Acharya <sacharya@redhat.com>
Date: Mon, 3 Jun 2019 18:05:24 +0530
Subject: [PATCH 186/192] cli: Remove-brick warning seems unnecessary

As force-migration option is disabled by default,
the warning seems unnessary.

Rephrased the warning to make best sense out of it.

>fixes: bz#1712668
>Change-Id: Ia18c3c5e7b3fec808fce2194ca0504a837708822
>Signed-off-by: Shwetha K Acharya <sacharya@redhat.com>

backport of https://review.gluster.org/#/c/glusterfs/+/22805/

Bug: 1708183
Change-Id: Ia18c3c5e7b3fec808fce2194ca0504a837708822
Signed-off-by: Shwetha K Acharya <sacharya@redhat.com>
Reviewed-on: https://code.engineering.redhat.com/gerrit/173447
Tested-by: RHGS Build Bot <nigelb@redhat.com>
Reviewed-by: Sunil Kumar Heggodu Gopala Acharya <sheggodu@redhat.com>
---
 cli/src/cli-cmd-volume.c | 17 +++++++++--------
 1 file changed, 9 insertions(+), 8 deletions(-)

diff --git a/cli/src/cli-cmd-volume.c b/cli/src/cli-cmd-volume.c
index 564aef7..a42e663 100644
--- a/cli/src/cli-cmd-volume.c
+++ b/cli/src/cli-cmd-volume.c
@@ -2090,14 +2090,15 @@ cli_cmd_volume_remove_brick_cbk(struct cli_state *state,
             " on the volume.\nDo you want to continue?";
     } else if (command == GF_OP_CMD_START) {
         question =
-            "Running remove-brick with cluster.force-migration"
-            " enabled can result in data corruption. It is safer"
-            " to disable this option so that files that receive "
-            "writes during migration are not migrated.\nFiles "
-            "that are not migrated can then be manually copied "
-            "after the remove-brick commit operation.\nDo you "
-            "want to continue with your current "
-            "cluster.force-migration settings?";
+            "It is recommended that remove-brick be run with"
+            " cluster.force-migration option disabled to prevent"
+            " possible data corruption. Doing so will ensure that"
+            " files that receive writes during migration will not"
+            " be migrated and will need to be manually copied"
+            " after the remove-brick commit operation. Please"
+            " check the value of the option and update accordingly."
+            " \nDo you want to continue with your current"
+            " cluster.force-migration settings?";
     }
 
     if (!brick_count) {
-- 
1.8.3.1

